src/sql/magic.py

```diff
@@ -367,11 +373,34 @@ def interactive_execute_wrapper(**kwargs):
             alias=args.alias,
         )
         payload["connection_info"] = conn._get_curr_sqlalchemy_connection_info()
-        if args.persist:
+        if args.persist_replace and args.append:
+            raise exceptions.ValueError(
+                """You cannot simultaneously replace and append data to a dataframe;
+                  please choose to utilize either one or the other."""
+            )
+        if args.persist and args.persist_replace:
```

Comment:
what happens if we pass append and persist_replace? sounds like it should also fail, right? so it sounds like if more than one option is set in {persist, append, persist_replace} we should throw an error

Reply:
Yes, it will fail
<img width="912" alt="Screenshot 2023-05-24 at 3 47 41 PM" src="https://github.com/ploomber/jupysql/assets/123580782/3f21f238-0491-49c6-be62-802a0478bfed">

Reply:
alright, this makes sense